json-schemas/src/account-stats.json

```diff
@@ -27,17 +27,27 @@
         "messages.all.all.count": {
           "type": "number",
           "inclusiveMinimum": 0,
-          "description": "Total message count."
+          "description": "Total number of successful messages, summed over all message types and transports."
         },
         "messages.all.all.data": {
           "type": "number",
           "inclusiveMinimum": 0,
-          "description": "Total message size."
+          "description": "Total message size of all messages successfully sent, summed over all message types and transports."
         },
         "messages.all.all.uncompressedData": {
           "type": "number",
           "inclusiveMinimum": 0,
-          "description": "Total uncompressed message size (excluding any compression, e.g. delta compression: https://ably.com/documentation/realtime/channels/channel-parameters/deltas)."
+          "description": "Total uncompressed message size (excluding any compression, e.g. delta compression: https://ably.com/docs/channels/options/deltas)."
```

Comment:
We say "e.g. delta compression" quite frequently, but do we actually do any other type of compression?

Reply:
I've reworded